server/controllers/aidComments.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable camelcase */
+/* eslint-disable sort-keys */
+import {
+    INTERNAL_SERVER_ERROR, CREATED, NOT_FOUND, getStatusText
+} from 'http-status-codes';
+
+const db = require('./promise').AidDb;
+
+const Comments = {
+    async create(req, res) {
```

Comment:
What's does this function do?

Reply:
it handles the create, read and update for the comments

Reply:
Interesting

Reply:
This looks good to me